bazel/repositories.bzl

```diff
@@ -180,6 +180,18 @@ def envoy_api_deps(skip_targets):
         actual = "@googleapis//:http_api_protos_lib",
     )
 
+def abseil_deps(skip_targets):
+  if 'abseil' not in skip_targets:
+    native.git_repository(
+        name = "abseil",
```

Comment:
Nit: the upstream workspace name is `com_google_absl`. Bazel will warn if you use a different local name, and says this will eventually be an error.